packages/vaults/src/transaction/wallet/wallet_update_name.ts

```diff
@@ -9,8 +9,20 @@ import {TransactionRequest} from "../transaction_request";
 import {RequestMapperAbstract} from "../request_mapper";
 
 
+/**
+ * Interface for a transaction that updates the name of an existing wallet.
+ * This transaction can be created or approved by an admin.
+ * This transaction can be executed in a batch.
```

Comment:
This transaction can only be requested or approved by users with the admin role.
This transaction can be executed in a batch, which means that all transactions marked with a single batch_id are executed or rejected together.